common/components/contact/Contact.css

```diff
@@ -1,10 +1,25 @@
 /** @define Contact */
 
-.Contact-title {
+.Contact-text {
   color: var(--color-black);
   font-family: var(--font-family-title);
-  font-size: 48px;
-  line-height: 60px;
+  font-size: 38px;
+  line-height: 48px;
   margin-top: 52px;
   text-align: center;
+  word-wrap: break-word;
 }
+
+a {
```

Comment:
This is causing an error because you've used a plain `a` selector. It should be a class, something like `.Contact-link` should do it.